src/misc/unicode.cpp

```diff
@@ -754,12 +770,12 @@ static void warn_code_page(const uint16_t code_page)
 	LOG_WARNING("UNICODE: Requested unknown code page %d", code_page);
 }
 
-static bool wide_to_dos(const std::vector<uint16_t>& str_in, std::string& str_out,
-                        const DosStringConvertMode convert_mode,
-                        const UnicodeFallback fallback, const uint16_t code_page)
+static std::string wide_to_dos(const wide_string& str_in,
+                               const DosStringConvertMode convert_mode,
+                               const UnicodeFallback fallback,
+                               const uint16_t code_page)
```

Comment:
Just `str` instead of `str_in` (all `const` args are inputs, that's obvious).

Reply:
OK, changed everywhere.